course/flask/README.md

```diff
@@ -1,9 +1,11 @@
-
-
 #### Introduction to Flask Microframework for Web Development
 
-This is a basic introduction to flask intended to give us an understanding of the popular Model, Template, View web application code pattern. Flask has [an excellent tutorial](http://flask.pocoo.org/docs/0.10/).
+This is a basic introduction to Flask demonstrating the popular
+[Model, Template, View](https://docs.djangoproject.com/en/2.0/faq/general/#django-appears-to-be-a-mvc-framework-but-you-call-the-controller-the-view-and-the-view-the-template-how-come-you-don-t-use-the-standard-names)
```

Comment:
I guess there's some debate about whether Flask really even is an MVC at all.  I don't love linking to `django` here because I think it might distract/confuse some students who are new to the stuff (there's so much info about django, since it's a large project).

Could we just link to the Flask tutorial and drop the MVC language?

Reply:
@jayaddison The MTV thing is legacy content; I'm happy with dropping it from the lesson altogether. I don't think that abstraction is particularly useful for explaining things to new coders in the first place. I can rewrite the lesson to highlight how Flask allows us to glue disparate things together nicely, which is more in the spirit of the framework.